ui/src/components/charts/job-durations.tsx

```diff
@@ -142,8 +143,28 @@ export const JobDurations = ({
           ).durationMs
         : null;
 
+    // Calculate how many durations are non-null. This is needed for proper indexing in the tooltip,
+    // to render the total duration at the end of the tooltip.
+    const finishedJobs = [
+      analyzerDuration,
+      advisorDuration,
+      scannerDuration,
+      evaluatorDuration,
+      reporterDuration,
+    ].filter((duration) => duration !== null).length;
+
+    // Calculate the total duration
+    const totalDuration =
+      (analyzerDuration ?? 0) +
+      (advisorDuration ?? 0) +
+      (scannerDuration ?? 0) +
+      (evaluatorDuration ?? 0) +
+      (reporterDuration ?? 0);
```

Comment:
You should rather use the `createdAt` and `finishedAt` properties of the run object itself to get the total run duration.

Reply:
Oh, that's an interesting point. In that regard also the stacked-bar-chart visualization is wrong / misleading, as times do not all add up...